src/main/scala/hellotest/Main.scala

```diff
@@ -54,7 +54,7 @@ object Main:
     val ignoreList = readIgnoreFile(args.ignoreFilePath)
     
     // 2. filter ignored values and words that don't meet minimum length
-    val filteredInput = input // DO:
+    val filteredInput = input.filterNot(word => {ignoreList(word) || word.length < args.minLength}) // DO:
```

Comment:
I forgot to make a unit test for this. Can you turn this into a function and make a test for it?